src/index.js

```diff
@@ -6,7 +6,6 @@ const defaults = {
   urlRegularExpression: /(https?:\/\/[\S]+)|(www.[\S]+)/,
   normalizeRegularExpression: /(https?:\/\/[\S]+)|(www.[\S]+)/,
   normalizeUrlOptions: {
-    stripFragment: false,
```

Comment:
This is `false` by default in v4 and it's called `stripHash`:
https://github.com/sindresorhus/normalize-url/tree/v4.5.0#striphash